### ISSUE TITLE
Travis CI could not find your repository

### ISSUE THREAD

**Issue by zeybek** [felixrieseberg/vsc-travis-ci-status#24]
```
"travis.repository": "GitHubProfileCard",
 "travis.username": "hyp3ri0nn",
```
I have added this lines to `settings.json` but didn't work, how we should write it ?

**Comment by magnus-trent** [comment]
Same here. I use Travis Pro and have added my Token and everything required yet it stills says cannot for TheGrimSilence/[repo]

**Comment by magnus-trent** [comment]
Just tested `Open in Travis CI` and it took me to the correct repository on travis-ci.com

**Comment by zeidlos** [comment]
Same problem here.

**Comment by zeybek** [comment]
This extension works only with `travis-ci.org` not `travis-ci.com`

**Comment by ianpaschal** [comment]
This should be re-opened since Travis CI will be discontinuing .org in favor of .com. From Travis CI's own blog:

> Over the next several months, we’ll be migrating all travis-ci.org repositories and customers to travis-ci.com.

> If you’re activating a new repository – either public or private – on Travis CI, please use travis-ci.com.

[Source](https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps)

This whole extension is soon to be worthless and non-functional unless this bug is addressed.

**Comment by psychelzh** [comment]
Yes, this issue should be re-opened. There is no reason why this extension only works for `.org`, regardless of the migration from `.org` to `.com`.

**Comment by Moser-ss** [comment]
This feature already exists in the code. It is a matter to add the ` "travis.pro": true`in  `./.vscode/settings.json`

**Comment by fietenoer** [comment]
I will copy my comment from another issue:

You need to set `"travis.pro": true` in your settings.json.

Next you need to set `"travis.github_oauth_token": "<TOKEN>"`. (only applies for private repos)
You can get your  token from `https://github.com/settings/tokens` where you need to generate one with the repo permissions (Full control of private repositories). Note that it might be possible to give less permissions, however I haven't tested that yet.

**Comment by TheRealWaldo** [comment]
I've yet to get this working....  I've tried everything listed in this topic and still get the same message no matter what I do.  I have verified the URLs do work from the browser; but something in the plugin is not working.
